scripts/recan_bundle_tdr.py

```diff
@@ -245,6 +245,9 @@ def to_json_row(self) -> JSON:
         return dict(super().to_json_row(),
                     file_id=drs_uri(self.file_manifest_entry['drs_path']),
                     descriptor=dict(tdr.Checksums.from_json(self.file_manifest_entry).to_json(),
+                                    # FIXME: Support non-null DRS URIs in file descriptors
+                                    #        https://github.com/DataBiosphere/azul/issues/3631
+                                    drs_uri=None,
```

Comment:
This looks as if it makes every file in the can is now a phantom file. That
s not representative of reality. Then default case should be normal, internal file.